cloud-config/deploy-run-scripts/ami-preapp.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Yea, that was a late addition.  I made the demo first, then es node, then frontend.  The separation became important after a first organization. 

I can fix up the names, but this will be cleaned up with ansible.    And changing them would require retesting...which I may have to do anyway.